### PR TITLE
Basic noya-designsystem components 

### PR DESCRIPTION
**Pull request by michalsek**

### CODE REVIEW

File: packages/app-mobile/src/containers/Canvas.tsx [hunk 1]
```diff
@@ -290,15 +290,16 @@ const Canvas: React.FC<{}> = () => {
...
+  // const interactionView = useMemo(
+  //   () => (
+  //     <InteractionView>
+  //       <FpsCounter visible />
+  //       <Interaction>{state.interactionState.type}</Interaction>
+  //     </InteractionView>
+  //   ),
+  //   [state.interactionState.type],
+  // );
+  // {interactionView}
```
Comment (michalsek) [inline packages/app-mobile/src/containers/Canvas.tsx:302]: cleanup

File: packages/app-mobile/src/containers/Canvas.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ import {
 import { AffineTransform, Point, createRect } from 'noya-geometry';
 import { Selectors, getCurrentPage } from 'noya-state';
 import { useCanvasKit, useFontManager } from 'noya-renderer';
-import { FpsCounter } from '../components/FPSCounter';
+// import { FpsCounter } from '../components/FPSCounter';
```
Comment (michalsek) [inline packages/app-mobile/src/containers/Canvas.tsx:18]: cleanup

File: packages/noya-designsystem/src/components/Tooltip/Tooltip.native.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React, { memo } from 'react';
```
Comment (michalsek) [inline packages/noya-designsystem/src/components/Tooltip/Tooltip.native.tsx:1]: remove for now?

File: packages/noya-designsystem/src/components/internal/TextInput/TextInput.native.tsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+        return;
+      }
+
+      onSubmit(submissionValue);
+      setInternalValue(latestValue.current);
+    },
+    [value, internalValue, allowSubmittingWithSameValue, onSubmit],
+  );
+
+  const hnadleSubmit = useCallback(() => {
```
Comment (dabbott) [inline packages/noya-designsystem/src/components/internal/TextInput/TextInput.native.tsx:93]: Typo on `handle`

**Review by dabbott — APPROVED** [review]
This looks really great! It's much easier to understand design system components at a glance now that they're broken down a bit more

**Pull request merged**